package.json

```diff
@@ -0,0 +1,11 @@
+{
+	"name": "@moai/root",
+	"lint-staged": {
+		"*.{ts,tsx,js,css,json}": "prettier --write"
+	},
+	"devDependencies": {
+		"husky": "^6.0.0",
+		"lint-staged": "^10.5.4",
+		"prettier": "^2.2.1"
+	}
```

Comment:
@lqt93 I'm sorry but can you use exact version? It's just I want to manually control the dependencies upgrading